src/main/java/tr/com/orioninc/laborant/service/LabService.java

```diff
@@ -87,17 +87,17 @@ public String getALlLabsStatus(){
                             words.add(tokenizer.nextToken());
                         outputArray.add(words);
 
-                        System.out.println(currentLine);
+                        log.info(currentLine); // I dont know what this info prints.
                     }
-                    System.out.println(outputArray);
+                    log.info(outputArray); // I dont know what this info prints.
```

Comment:
Please debug the program and resolve what this log prints. Depending on information you can improve or remove the log line.